.github/workflows/build-deploy.yml

```
@@ -0,0 +1,53 @@
+  push:
+    branches:
+      - main  # Trigger the deployment only on pushes to the 'gcp' branch.
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Step 1: Checkout the repository
+    - name: Checkout code
+      uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)